keepassxc-browser/content/pwgen.js

```diff
@@ -137,14 +141,13 @@ kpxcPasswordGenerator.fill = function(elem, password) {
     // Fill next password field if found
     if (kpxc.inputs.length > 0) {
         const index = kpxc.inputs.indexOf(elem);
-        const nextField = kpxc.inputs[index + 1];
-
-        kpxcPasswordDialog.nextField =
-            nextField && nextField.getLowerCaseAttribute('type') === 'password' ? nextField : undefined;
-        if (kpxcPasswordDialog.nextField) {
-            kpxcPasswordDialog.nextField.value = password;
-            kpxcPasswordDialog.nextField.dispatchEvent(new Event('input', { bubbles: true }));
-            kpxcPasswordDialog.nextField.dispatchEvent(new Event('change', { bubbles: true }));
+        const next = kpxc.inputs[index + 1];
```

Comment:
Need to check the length of the inputs first, you just check for >0, or does js just return null and not complain about out of bounds?

Reply:
Right. Let's add some extra checks here just in case.

Reply:
Actually `.at()` and [] operator both returns `undefined` if there's no value.